src/vs/workbench/contrib/debug/browser/media/debugViewlet.css

```diff
@@ -120,18 +120,19 @@
 	text-overflow: ellipsis;
 }
 
-.debug-pane .debug-call-stack .thread > .state,
-.debug-pane .debug-call-stack .session > .state {
-	display: flex;
-	align-items: center;
-	text-align: right;
+.debug-pane .debug-call-stack .thread > .label,
+.debug-pane .debug-call-stack .session > .label {
 	overflow: hidden;
 	text-overflow: ellipsis;
-	padding: 0 10px;
+	margin: 0 10px;
 	text-transform: uppercase;
+	align-self: center;
+	font-size: 0.8em;
+	background-color: rgba(136, 136, 136, 0.27);
```

Comment:
You are hard coding the color inside the CSS. This is not a good approach since I think this is themable.
I think you should simple fix the rules here to take into account your removal of `.state`
https://github.com/microsoft/vscode/blob/2f63aa9e559089436aa237f9c256ebad56913e42/src/vs/workbench/contrib/debug/browser/debug.contribution.ts#L649

Reply:
Right. I'll fix this. Thanks.